src/app/profile/views/ens_manager.nim

```diff
@@ -93,6 +93,17 @@ QtObject:
     result.pendingUsernames = initHashSet[string]()
     result.setup
 
+  proc getFirstEnsUsername(self: EnsManager): string {.slot.} =
+    if self.usernames.len > 0:
+      return self.usernames[0]
+    return ""
```

Comment:
This is needed because if a user hasn't set a "preferred username" (remember accounts can have multiple ens names), we need to decide for them.